sdk/openai/openai/review/openai-models.api.md

```diff
@@ -850,6 +852,13 @@ export type OnYourDataVectorizationSourceType = string;
 // @public
 export type OnYourDataVectorizationSourceUnion = OnYourDataEndpointVectorizationSource | OnYourDataDeploymentNameVectorizationSource | OnYourDataModelIdVectorizationSource | OnYourDataVectorizationSource;
 
+// @public
+export class OpenAIError extends RestError {
```

Comment:
We probably don't need to export the class when `OpenAIErrorModel` is exported. Do we expect customers to construct objects of `OpenAIError` directly?

Reply:
That's a good point, it does not need to be exported.